games-board/netmaumau/netmaumau-0.16-r1.ebuild

```diff
@@ -32,6 +32,10 @@ DEPEND="
 
 S=${WORKDIR}/${P}-client
 
+src_prepare() {
+	epatch "${FILESDIR}/connect-to-old-servers.patch"
```

Comment:
We usually prepend ```${P}``` to the patch name, so it's easy to figure out which package and version the patch applies to.